app/views/guide_to_resetting_windows_laptops_and_tablets/unlock_recovery_mode.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render partial: 'guide_contents', locals: { index: 4 } %>
+
+    <p class="govuk-body">To enable the recovery partition, log out as a local user and log in as the administrator.</p>
+    
+    <h3 class="govuk-heading-m">
+      Administrator login details
+    </h3>
+    
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Username: .\localadmin</li>
+      <li>Password: The admin user password retrieved through <%= govuk_link_to 'step 2', devices_guide_to_resetting_windows_laptops_and_tablets_get_local_admin_and_bios_passwords_path %> of this guide</li>
+    </ul>
+    
+    <p class="govuk-body">Once logged in as administrator:</p>
+    
+    <ol class="govuk-list govuk-list--number">
```

Comment:
Because these are a set of steps performed in order, do you want a `<ul>` instead?